charts/fwd-proxy/chart/values.yaml

```
@@ -42,6 +43,91 @@ nginx:
     #   cpu: 100m
     #   memory: 128Mi
 
+caddy:
+  enabled: true
+  image:
+    repository: caddy
+    pullPolicy: IfNotPresent
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: "2.5.2-alpine"
+
+  securityContext: {}
+    # capabilities:
+    #   drop:
+    #   - ALL
+    # readOnlyRootFilesystem: true
+    # runAsNonRoot: true
+    # runAsUser: 1000
+
+  resources: {}
+    # We usually recommend not to specify default resources and to leave this as a conscious
+    # choice for the user. This also increases chances charts run on environments with little
+    # resources, such as Minikube. If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+    # requests:
+    #   cpu: 100m
+    #   memory: 128Mi
+
+  caddyfile: |
+    {
+        debug
+        auto_https disable_redirects
+        local_certs
+        skip_install_trust
+    }
+
+    :80 {
+      respond / 404
+      respond /healthz 200
+    }
+
+    {{- $registries := list "registry-1.docker.io" "ghcr.io" "k8s.gcr.io" "quay.io" "gridopsingularity.azurecr.io" }}
+    {{- range $registries }}
+    {{ . }}:443 {
```

I guess for now we should strive for feature parity with Nginx and also listen on http:
```suggestion
    http://{{ . }}, https://{{ . }} {
```